src/components/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,272 @@
+@import "../../constants/style";
+
+.column__header-color-picker-dropdown {
+  z-index: 220;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  //gap: $spacing--xxs;
+  //padding-top: $spacing--xxs;
+  padding-bottom: $spacing--xxs;
+  box-shadow: 0 0 20px rgba(55, 62, 79, 0.12);
+  //background-color: #ffffff;
+  background-color: $gray--300;
+
+  //width: 40px;
+
+  //box-shadow:
+  //  0 4px 6px -1px rgb(0 0 0 / 0.1),
+  //  0 2px 4px -2px rgb(0 0 0 / 0.1);
+  list-style: none;
+  //padding: 10px;
+  //padding-top: 30px;
+  padding-left: 0;
+  margin-top: auto;
+  border-radius: 24px;
+  //position: inherit;
+  //left: 30px;
+
+  &:hover {
+    background-color: $brighten--slightly;
+  }
+}
+
+.column__header-color-picker-dropdown > li > button {
+  cursor: pointer;
+  border: none;
+  background: none;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.column__header-color-picker-dropdown > li {
+  all: unset;
```

Comment:
Why is it necessary?

Reply:
i thought it was necessary to reset the list elements properties to their default values here but it actually works without, good point! :)